server/index.js

```diff
@@ -1,18 +1,18 @@
+require('newrelic');
+require('dotenv').config();
 const express = require('express');
 const bodyParser = require('body-parser');
 const path = require('path');
 
+
 const app = express();
 const detail = require('./routes/detail.js');
-const userRequest = require('./routes/userRequest.js');
-require('dotenv').config();
 
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
-app.use('/detail', detail);
-app.use('/userRequest', userRequest);
-app.use(express.static(path.join(__dirname, '/../client/dist/')));
-
+app.use('/homes/:home', express.static(path.join(__dirname, '/../client/dist/')));
```

Comment:
```suggestion
app.use(['/homes/:home', '/addresses/:address'], express.static(path.join(__dirname, '/../client/dist/')));
```
These two lines can be combined